src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/ProjectFileReaderTests.cs

```diff
@@ -243,5 +243,63 @@ public void ProjectFileReader_FindAssemblyNameShouldBeNullWhenNotAvailable()
             var result = new ProjectFileReader().ReadProjectFile(xDocument, "");
             result.AssemblyName.ShouldBeNull();
         }
+
+        [Theory]
+        [InlineData("netcoreapp2.0;net461;netstandard2.0")]
+        [InlineData("netcoreapp1.1;net45")]
+        [InlineData("netcoreapp2.1")]
+        public void ProjectFileReader_ShouldParseWhenMultipleFrameworks(string target)
+        {
+            var xDocument = new XDocument();
+
+            if (target.Contains(";"))
+            {
+                xDocument = XDocument.Parse($@"
+<Project Sdk=""Microsoft.NET.Sdk"">
+    <PropertyGroup>
+        <TargetFrameworks>{target}</TargetFrameworks>
```

Comment:
Maybe it is a bit hacky but you could just ad the 's' to `<TargetFramework> conditionally instead of having to place the whole csproj file here twice. Something like 
``` xml
<{targetFrameworkName}>{target}</{targetFrameworkName}>
```

It would clean up this test a bit

Reply:
Sorry, I didn't know how to do conditional interpolation.  But I do now.